features/topology-map/org.opennms.features.topology.app/src/main/java/org/opennms/features/topology/app/internal/gwt/client/ui/CustomDisplay.java

```diff
@@ -28,13 +28,13 @@
 
 package org.opennms.features.topology.app.internal.gwt.client.ui;
 
+import java.util.Collection;
```

Comment:
We really need to get IDEA and Eclipse to agree on where `java.util.*` classes go.  :D

Reply:
For exactly this reason, there is custom configuration you must apply in order to use IntelliJ accordingly to prevent these re-imports. See https://github.com/opennms/ide-setup and for even more details https://wiki.opennms.org/wiki/IDEA_and_OpenNMS

However, when I first used IntelliJ code were already contributed, not following the eclipse import guides. Therefore occasionally if the contributor is not carefully enough, you get re-imports. 

That is why I always recommend `git add -i -p` instead of `git add .` to check what you are committing. In addition `git checkout -i -p` helps you getting rid of those re-imports.